test/python/algorithms/test_optimizer_de.py

```diff
@@ -0,0 +1,64 @@
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test of differential evolution optimizer."""
+
+import unittest
+from test.python.algorithms import QiskitAlgorithmsTestCase
+from scipy.optimize import rosen
+import numpy as np
+
+from qiskit import BasicAer
+from qiskit.algorithms import VQE
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.utils import QuantumInstance, algorithm_globals
+from qiskit.opflow import PauliSumOp
+from qiskit.algorithms.optimizers import DE
+
+
+class TestOptimizerDE(QiskitAlgorithmsTestCase):
```

Comment:
All optimizers are also tested on a small example in `test/python/algorithms/test_optimizers.py`. Could you add one test case for this optimizer?

Reply:
Ok added - let me know if this works